.rubocop.yml

```diff
@@ -21,6 +21,9 @@ AllCops:
 Style/Documentation:
   Enabled: false
 
+Style/NegatedIf:
+  Enabled: false
```

Comment:
Let's avoid making changes to Rubocop as it defeats the purpose of having shared code style conventions. Adapting the code to fit the team's code style is best.

Reply:
Fixed in https://github.com/Goose97/google-scraper-ruby/pull/20/commits/7818449712a3ebb39bbdc369b1e88555cdf9ef4c